package.json

```diff
@@ -97,13 +94,16 @@
     "eslint-plugin-prettier": "^3.0.1",
     "husky": "^4.2.3",
     "istanbul": "^0.4.3",
+    "jest": "^25.1.0",
+    "lodash": "^4.17.21",
     "mocha": "^7.1.0",
```

Comment:
We are not using this package, remove please :) 
Same comment for all unused packages